ocaml/xapi/message_forwarding.ml

```diff
@@ -1683,7 +1684,20 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
       info "VM.migrate_send: VM = '%s'" (vm_uuid ~__context vm);
       let local_fn = Local.VM.migrate_send ~vm ~dest ~live ~vdi_map ~vif_map ~options in
       let forwarder =
-        if Xapi_vm_lifecycle.is_live ~__context ~self:vm then forward_vm_op else
+        if Xapi_vm_lifecycle.is_live ~__context ~self:vm then
+          let host = List.assoc Xapi_vm_migrate._host dest |> Ref.of_string in
+          if Db.is_valid_ref __context host then
+            (* Intra-pool: reserve resources on the destination host, then
+             * forward the call to the source. *)
+            let snapshot = Helpers.get_boot_record ~__context ~self:vm in
+            (fun ~local_fn ~__context ~vm f ->
```

Comment:
Can you rename `f` with `op` here and in the next line? I think it is clearer what it is about and it is more consistent with the way it is called in other parts of the code